tests/robot-tests/tests/admin/bau/create_methodology_amendment.robot

```diff
@@ -93,6 +94,11 @@ Verify the editable content is as expected
 Approve the Methodology
     user approves methodology for publication    ${PUBLICATION_NAME}    ${PUBLICATION_NAME} - first methodology version
 
+Visit methodology on public side and check it is visible on 'all methodologies' page
+    user verifies methodology is visible on public methodology page    %{TEST_THEME_NAME}    %{TEST_TOPIC_NAME}
```

Comment:
I think we can refactor similar checks in `bau\publish_methodology.robot` to use this now.